packages/components/message/src/method.ts

```diff
@@ -41,6 +41,9 @@ const normalizeOptions = (params?: MessageParams) => {
     ...options,
   }
 
+  if (!normalized.appendTo) {
+    normalized.appendTo = document.body
+  }
   if (isString(normalized.appendTo)) {
```

Comment:
Maybe use `else if` link,one judgment can be reduced.

Reply:
You are right.